lib/topolys/model.rb

```diff
@@ -1028,7 +1028,7 @@ class Wire < Object
     attr_reader :plane
 
     # @return [Vector3D] outward normal of this wire's plane
-    attr_reader :outward_normal
+    attr_reader :normal
```

Comment:
@rd2 this was the problem, I just couldn't decided whether to name this `normal` or `outward_normal` and left it in an inconsistent state.  `wire.normal` should work now and it is cached so will be a bit faster than `wire.plane.normal`